launch/hpp_agimus_controller.launch

```diff
@@ -1,7 +1,7 @@
 <launch>
 
-    <include file="$(find agimus_controller)/launch/hpp_bringup.launch"/>
+    <node pkg="agimus_controller" name="hpp_corbaserver" type="hpp_corbaserver" />
```

Comment:
Same principle as before, create a group to affect all sub part into a single namespace.